packages/language-support/src/formatting/formattingHelpersv2.ts

```diff
@@ -79,19 +79,24 @@ const traillingCharacters = [
   CypherCmdLexer.RBRACKET,
 ];
 
+// TODO: This function should probably not exist; we're not really fans of
+// shuffling around the AST like we're doing right now...
```

Comment:
Not related to the PR but just for your own understanding: we don't have an AST, we have a parsing tree (CST)